python/dgl/distributed/rpc.py

```diff
@@ -143,8 +299,38 @@ def service_id(self):
             raise DGLError('Request class {} has not been registered as a service.'.format(cls))
         return sid
 
+    @property
+    def server_id(self):
+        """Get server ID"""
+        return self._server_id
+
+    @property
+    def client_id(self):
+        """Get client ID"""
+        return self._client_id
+
+    @property
+    def msg_seq(self):
+        """Get msg_seq"""
+        return self._msg_seq
+
+    @server_id.setter
+    def server_id(self, server_id):
+        """Set server ID"""
+        self._server_id = server_id
+
+    @client_id.setter
+    def client_id(self, client_id):
+        """Set client ID"""
+        self._client_id = client_id
+
+    @msg_seq.setter
+    def msg_seq(self, msg_seq):
+        """Set msg_seq"""
+        self._msg_seq = msg_seq
```

Comment:
Why adding these properties?

Reply:
These properties will be used in send_response.